src/core/worker.js

```diff
@@ -591,7 +597,8 @@ var WorkerMessageHandler = {
           password: source.password,
           length: fullRequest.contentLength,
           disableAutoFetch: disableAutoFetch,
-          rangeChunkSize: source.rangeChunkSize
+          rangeChunkSize: source.rangeChunkSize,
+          filename: filename
```

Comment:
Yes, you should add one. We have not been too consistent with this, but for new or changed code we do add this.